meetup/tests.py

```diff
@@ -47,65 +91,62 @@ def test_meetup_link_is_unique(self):
-    self.assertEquals(place, meetup.pizza_places.all()[0])
+    meetup= Meetup.objects.create(name="Meetup1", meetup_link='http://meetup.com/some-meetup')
+    place = meetup.pizza_places.create(name="Pete Zazz", yelp_link='https://www.yelp.com/biz/prince-st-pizza-new-york')
+    self.assertEquals(place, meetup.pizza_places.first())
 
 class TestMeetupModelValidations(TestCase):
 
   def setUp(self):
-    self.patcher = patch('meetup.models.MeetupApiLookupAgent')
+    self.patcher = patch('meetup.models.MeetupApi')
     self.mock_agent = self.patcher.start()
-    self.addCleanup(self.patcher.stop)
 
   def test_meetup_raises_error_on_invalid_url(self):
-    meetup= Meetup(name="Meeetup1", meetup_link='hi/ok/what')
+    meetup= Meetup(name="Meetup1", meetup_link='hi/ok/what')
     self.assertRaises(ValidationError, meetup.full_clean)
 
   def test_error_raised_if_link_does_not_point_to_meetupdotcom(self):
-    meetup= Meetup(name="Meeetup1", meetup_link='http://www.example.com/')
+    meetup= Meetup(name="Meetup1", meetup_link='http://www.example.com/')
     self.assertRaises(ValidationError, meetup.full_clean)
 
   def test_error_raised_if_no_urlname_in_meetup_url(self):
-    meetup= Meetup(name="Meeetup1", meetup_link='http://www.meetup.com/')
+    meetup= Meetup(name="Meetup1", meetup_link='http://www.meetup.com/')
     self.assertRaises(ValidationError, meetup.full_clean)
 
   def test_error_raised_if_no_trailing_slash_in_meetup_url(self):
-    meetup= Meetup(name="Meeetup1", meetup_link='http://www.meetup.com/hackerhours')
+    meetup= Meetup(name="Meetup1", meetup_link='http://www.meetup.com/hackerhours')
     self.assertRaises(ValidationError, meetup.full_clean)
 
   def test_error_raised_if_multiple_urlnames(self):
-    meetup= Meetup(name="Meeetup1", meetup_link='http://www.meetup.com/hackerhours/events/')
+    meetup= Meetup(name="Meetup1", meetup_link='http://www.meetup.com/hackerhours/events/')
     self.assertRaises(ValidationError, meetup.full_clean)
 
   def test_meetup_url_with_urlname_and_trailing_slash_passes(self):
-    meetup= Meetup(name="Meeetup1", meetup_link='http://www.meetup.com/hackerhours/')
+    meetup= Meetup(name="Meetup1", meetup_link='http://www.meetup.com/hackerhours/')
     errors_raiesed_by_meetup = meetup.full_clean()
     self.assertTrue(errors_raiesed_by_meetup == None)
 
   def test_url_with_dashes_in_urlname_passes(self):
-    meetup = Meetup(name="Meeetup1", meetup_link='http://www.meetup.com/papers-we-love/')
+    meetup = Meetup(name="Meetup1", meetup_link='http://www.meetup.com/papers-we-love/')
     errors_raiesed_by_meetup = meetup.full_clean()
     self.assertIsNone(errors_raiesed_by_meetup)
 
   def test_non_real_meetup_raises_validation_error(self):
-    self.mock_agent.return_value.is_real_meetup.return_value = False
-    meetup = Meetup(name="Meeetup1", meetup_link='http://www.meetup.com/la-la-la/')
+    self.mock_agent.return_value.url_exists.return_value = False
```

Comment:
Now that this class has been renamed, you may want to clean up this instance variable name, too.